docs/02-app/02-api-reference/05-next-config-js/ppr.mdx

```diff
@@ -83,7 +83,7 @@ export default function Page() {
 
 ### Enabling PPR (Version 14)
 
-For version 14, you can enable it by adding the [`ppr`](/docs/app/api-reference/next-config-js/ppr) option to your `next.config.js` file. This will apply to all routes in your application:
+For version 14, you can enable it by adding the [`ppr`](/docs/app/api-reference/next-config-js/ppr) option to your `next.config.js` file. You must also be on version 14 canary to enable PPR. This will apply to all routes in your application:
```

Comment:
```suggestion
For version 14, you can enable it by adding the [`ppr`](/docs/app/api-reference/next-config-js/ppr) option to your `next.config.js` file. You must also be on canary version to enable PPR. This will apply to all routes in your application:
```